hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ScmConfigKeys.java

```diff
@@ -48,6 +48,12 @@
       = "dfs.container.ratis.num.write.chunk.threads";
   public static final int DFS_CONTAINER_RATIS_NUM_WRITE_CHUNK_THREADS_DEFAULT
       = 60;
+  public static final String
+          DFS_CONTAINER_RATIS_NUM_WRITE_CHUNK_EXECUTOR_QUEUE_LIMIT
+          = "dfs.container.ratis.num.write.chunk.executor.queue.limit";
```

Comment:
`num` stands for "number of ..." (eg. in `dfs.container.ratis.num.write.chunk.threads` it is "number of ... threads").  It seems to be unnecessary in this property name, "... queue limit" seems better than "number of ... queue limit".